language/libra-vm/src/libra_transaction_executor.rs

```diff
@@ -724,6 +724,7 @@ fn convert_txn_args(args: &[TransactionArgument]) -> Vec<Value> {
             TransactionArgument::Address(a) => Value::address(*a),
             TransactionArgument::Bool(b) => Value::bool(*b),
             TransactionArgument::U8Vector(v) => Value::vector_u8(v.clone()),
```

Comment:
Just to double check, what do we plan to do with `TransactionArgument::U8Vector` once `TransactionArgument::Vector` gets fully supported? Shall we remove it eventually?

Reply:
We should keep it because it is much more efficient that a vector a tagged bytes